packages/vue/src/components/authenticator.vue

```diff
@@ -80,19 +80,29 @@ let unsubscribeHub: ReturnType<typeof listenToAuthHub>;
 const { state, send } = useActor(service);
 useAuth(service);
 
+const hasInitialized = ref(false);
+
+service.subscribe((newState) => {
+  if (newState.matches('setup')) {
+    if (!hasInitialized.value) {
```

Comment:
Nit: Think this can be combined with ln 86 in to one conditional statement

Reply:
Yep :p

Reply:
Done in [924e906](https://github.com/aws-amplify/amplify-ui/pull/1580/commits/924e9066bdf25cdce7f9b60ded3bf6376df0a60c)